azavea/us-cd-population-map
# Add support for Netlify deployments

This change set includes a variety of different changes to prepare the SPA for deployment via Netlify.

- [x] Connect Netlify URL to `population-map.districtbuilder.org` domain
- [x] Reset the main deployment branch in the Netlify configuration to `master`

Fixes https://github.com/azavea/operations/issues/466

---

**Testing**

- Ensure `npm run build` output contains no linter warnings
- Ensure the [application](https://deploy-preview-17--db-population-map.netlify.app/) functions as expected when served via Netlify
